test/pp_api_test/test_tokenizer.cc

```diff
@@ -346,7 +346,7 @@ TEST(OrtxTokenizerStreamTest, Llama2Tokenizer) {
   // validate tokenizer is not null
   EXPECT_TRUE(tokenizer != nullptr);
 
-  std::vector<std::string_view> input = {"This is a test and the second one. "};
+  std::vector<std::string_view> input = {"This is a test and the second one is in German. Ich liebe München!"};
```

Comment:
added test case for previously problematic char like ü that would cause the "UnicodeDecodeError: 'utf-8' codec can't decode byte" issue.